### PR TITLE
tidbits

### PR DESCRIPTION
**Pull request by Dylan-DPC**
Made some changes as suggested by @stjepang in #58

### CODE REVIEW

File: src/in-depth/human-communication.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Use it for, well, informative output.
 (Some applications that lean towards a more quiet output style
 might only show warnings and errors by default.)
 
-Additionally it's always a good idea to use similar prefixes
+Additionally. it's always a good idea to use similar prefixes
```
Comment (killercup) [inline src/in-depth/human-communication.md]: This should be a comma, right?

File: src/in-depth/machine-communication.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ In fact, this is a sentence from the [Unix philosophy]:
 
 [Unix philosophy]: https://en.wikipedia.org/wiki/Unix_philosophy
 
-If our programs fulfil this expectation,
+If our programs fulfill this expectation,
```
Comment (killercup) [inline src/in-depth/machine-communication.md]: TIL 'fulfil' is the British spelling

File: src/tutorial/packaging.md [hunk 3]
```diff
@@ -161,7 +161,7 @@ automatically?
 
 ### Building binary releases on CI
 
-If your tool is open source,
+If your tool is open sourced
```
Comment (killercup) [inline src/tutorial/packaging.md]: Hm, not sure I'd make this a past participle. My goal was saying being "open source" is an attribute of the code. But this change reads fine, let's keep it!

File: src/tutorial/errors.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ Now, we were able to access content of the file,
 but we can't really do anything with it after the `match` block.
 For this, we'll need to somehow deal with the error case.
 The challenge is that all arms of a `match` block need to return something of the same type.
-But there's a need trick to get around that:
+But there's a neat trick to get around that:
```
Comment (killercup) [inline src/tutorial/errors.md]: That's really embarrassing

File: src/tutorial/testing.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # Testing
 
 Over the decades of people doing software development
-one truth has been found:
+a pattern has emerged:
```
Comment (killercup) [inline src/tutorial/testing.md]: Nooo this takes all the fun out of it, let's keep this

File: src/tutorial/testing.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 Untested software rarely works.
-(Many people would go so far and add
+(Many people would go that far and add
```
Comment (killercup) [inline src/tutorial/testing.md]: "so" is correct imho

File: src/in-depth/human-communication.md (hunk 1 above)
Comment (Dylan-DPC) [reply src/in-depth/human-communication.md]: yep, typo :smile:

File: src/tutorial/testing.md (hunk 5 above)
Comment (Dylan-DPC) [reply src/tutorial/testing.md]: sure

File: src/tutorial/testing.md (hunk 6 above)
Comment (Dylan-DPC) [reply src/tutorial/testing.md]: it is correct but the sentence looks odd

Comment (killercup) [reply src/tutorial/testing.md]: You're right, I only looked at the diff. How about:

> Many people would go as far as saying: "Most tested software doesn't work either."

**Comment by killercup** [comment]
Thanks a lot!

**Pull request merged**